src/hera/workflows/_mixins.py

```diff
@@ -1272,3 +1277,93 @@ def _check_enabled(cls, values):
         if not global_config.experimental_features[cls._flag]:
             raise ValueError(cls._experimental_warning_message.format(cls, cls._flag))
         return values
+
+
+class EasyOutputsMixin(BaseMixin):
```

Comment:
I suggest adding a `subtype` field to this, so that child classes such as Step and Task can set it. That way this does not have to import `Step` or `Task` for type checking, and can rely on the `subtype` field to sub in the right string the same way `TemplateInvocatorSubNodeMixin` does it